<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rename</class>
 <widget class="QMainWindow" name="Rename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Rename</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tblRename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>921</width>
      <height>391</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>12</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>175</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Image Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Product ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Brand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Name</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnInputDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/image.svg</normaloff>Resources/image.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOutputDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/folder.svg</normaloff>Resources/folder.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutputDir">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>571</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select an output directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInputDir">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>571</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select an input directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddItem">
    <property name="geometry">
     <rect>
      <x>728</x>
      <y>425</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUploadList">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>425</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Data from File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCopyRename">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>470</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy and Rename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>425</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
